testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/contextdata/ClientInterceptor.java

```diff
@@ -42,20 +41,25 @@ public ClientInterceptor() {
 
   @Override
   public void handleInvocation(EJBClientInvocationContext context) throws Exception {
-    HashSet<String> keys = new HashSet<>();
-    keys.add("data1");
-    keys.add("data2");
-    context.getContextData().put("jboss.returned.keys", keys); //TODO: replace with call to org.jboss.ejb.client.EJBClientInvocationContext#addReturnedContextDataKey
-    context.getContextData().put("clientData", DATA);
+
+      logger.debugf("%s: ContextData: %s", "handleInvocation", context.getContextData());
+      ((UseCaseValidator)context.getParameters()[0]).test(UseCaseValidator.InvocationPhase.CLIENT_INT_HANDLE_INVOCATION, context.getContextData());
```

Comment:
the above `context.getParameters()` call can return null, and when accessing its 0th element, NPE is triggered.

The test ejb remote interface `TestRemote` contains 2 methods, one has param `UseCaseValidator` and the other does not. When the no-param method `invoke()` is called, the above NPE condition occurred:

`org.jboss.as.test.integration.ejb.remote.contextdata.ClientInterceptorReturnDataEjbOverHttpTestCase#testInvokeWithClientInterceptorData`, around line 110.

Reply:
@bmaxwell did you get a chance to look at it?